src/github.com/getlantern/flashlight/pac.go

```diff
@@ -41,16 +44,33 @@ func setUpPacTool() error {
 	return nil
 }
 
+func setProxyAddr(addr string) {
```

Comment:
Oh unfortunately through my testing browser (at least Chrome and Firefox) will not retrieve PAC file again until next launch, or when network status changed. This makes dynamically changed PAC nearly useless for those who keeps browser window open (including me).